### PR TITLE
Add support for Razer Viper Ultimate

### PR DESCRIPTION
**Pull request by octopusSD**
* Add support Razer Viper Ultimate - Wired (0x007A)
* Add support Razer Viper Ultimate - Wireless (0x007B)

Support for RGB charging dock (0x007E) not included.

### CODE REVIEW

**Comment by octopusSD** [comment]
Linked to #987

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -1287,6 +1287,112 @@ def _resume_device(self):
...
+        Get the last known brightness and then set the brightness
+        """
+        logo_brightness = self.suspend_args.get('brightness', 100)[0]
+
+        self.disable_notify = True
+        _da_set_logo_brightness(self, logo_brightness)
+        self.disable_notify = False
+
+
+class RazerViperUltimateWireless(__RazerDeviceBrightnessSuspend):
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py:1343]: You can make this class inherit `RazerViperUltimateWired` and just overwrite the values that are different on the Wireless version (such as `USB_PID`, `EVENT_FILE_REGEX`, etc)

**Review by z3ntu — CHANGES_REQUESTED** [review]
LGTM, just a small comment. Thanks for the PR!

**Comment by z3ntu** [comment]
Merged as 218bf33 . Thank you for the PR!

**Pull request closed without merging**